webcompat/templates/home-page/form.html

```diff
@@ -62,7 +62,8 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
             </span>
           </div>
           <div class="wc-Form-group js-Form-group">
-            {{ form.steps_reproduce.label(class_='wc-Form-label') }} {{ form.steps_reproduce(class_='wc-Form-item wc-Form-item--textarea js-autoExpand', rows=4) }}
+            <label class="wc-Form-label" for="steps_reproduce">How did you get there?</label>
+            <textarea class="wc-Form-item wc-Form-item--textarea js-autoExpand" id="steps_reproduce" name="steps_reproduce" rows="4" placeholder="Steps to reproduce the issue"></textarea>
```

Comment:
Can we include the pre-existing `steps_default` as a placeholder? I think it's good to give the user a hint to give us numbered steps (for example).